pkg/build/admission/jenkinsbootstrapper/admission.go

```diff
@@ -150,10 +153,14 @@ func (q *jenkinsBootstrapper) SetInternalKubeClientSet(c kclientset.Interface) {
 	q.serviceClient = c.Core()
 }
 
-func (a *jenkinsBootstrapper) SetOpenshiftClient(oclient client.Interface) {
+func (a *jenkinsBootstrapper) SetDeprecatedOpenshiftClient(oclient client.Interface) {
```

Comment:
@bparees I'm going to want to change this eventually.  There are integration tests that make sure it keeps working?

Reply:
@deads2k guess this change is not necessary for this PR, but I thought we want to mark that client as deprecated

Reply:
> @deads2k guess this change is not necessary for this PR, but I thought we want to mark that client as deprecated

Yes.  This is good and makes it obvious not to use it.  I'm just saying that in a future pull we should remove it and his usage is different than some.